src/components/ripple/ripple-manager.ts

```diff
@@ -0,0 +1,174 @@
+import {
+  ElementRef,
+} from '@angular/core';
+
+/** @internal */
```

Comment:
We discovered last week that we need to omit `@internal` (almost) everywhere in order for the offline compiler to work, so we're just forgoing it completely now. For things that shouldn't be part of the public API, just put "TODO: internal" and eventually we'll mark these things in our API docs. For properties/methods that are "internal", you can prefix them with an underscore but leave off the `private` modifier.